src/storage/mongodb/MongoDBStorage.ts

```diff
@@ -105,7 +105,6 @@ export default class MongoDBStorage {
       { fields: { deleted: 1, createdOn: 1 } },
       { fields: { issuer: 1, createdOn: 1 } },
       { fields: { userID: 1, issuer: 1 } },
-      { fields: { visualID: 1 }, options: { unique: true } }
```

Comment:
The goal is to have an index on that new field and to ensure its uniqueness, right? So you probably have removed that line by mistake.

Reply:
The unique index field need to be added after filling the visualID so we need to merge in 3 steps:

1. Remove unique index
2. fill the visualID
3. Re Add the index